package.json

```diff
@@ -13,13 +13,15 @@
     "ios": "npm run check-metro-bundler-port && react-native run-ios",
     "ipad": "npm run check-metro-bundler-port && react-native run-ios --simulator=\"iPad Pro (12.9-inch) (4th generation)\"",
     "ipad-sm": "npm run check-metro-bundler-port && react-native run-ios --simulator=\"iPad Pro (9.7-inch)\"",
-    "desktop": "node desktop/start.js",
+    "desktop": "export NODE_ENV=development && node desktop/start.js",
```

Comment:
Can you please move this down so that it's directly above the other desktop builds?